lib/chef/resource/plist.rb

```diff
@@ -85,7 +85,8 @@ class PlistResource < Chef::Resource # we name this PlistResource to avoid confu
         converge_if_changed :path do
           converge_by "create new plist: '#{new_resource.path}'" do
             file new_resource.path do
-              content {}.to_plist
```

Comment:
I think `content({}.to_plist)` should work here to avoid coercing to a block?

Reply:
Can confirm that works as well.